docs/sync.js

```diff
@@ -677,6 +678,14 @@ class ApiKeyAuth {
    * @returns {Promise<void>}
    */
   disable(id) {}
+
+  /**
+   * Deletes a user API key associated with the current user.
+   *
+   * @param {string} id - The id of the API key to delete.
+   * @returns {Promise<void>}
+   */
+  delete(id) {}
```

Comment:
I would like this method to be called `deleteApiKey`, but that would be a breaking change.

Reply:
Please create an issue :smile:

Reply:
Done https://github.com/realm/realm-js/issues/4322 :)